### ISSUE TITLE
Submit mappergpt to OAEI 2023

### ISSUE THREAD

**Issue by matentzn** [cmungall/gpt-mapping-manuscript#1]
See sister issue for OAK with the details https://github.com/INCATools/ontology-access-kit/issues/617

Info from email:

> The [OAEI Bio-ML track](https://www.cs.ox.ac.uk/isg/projects/ConCur/oaei/) will have its 2023 version, for providing a platform for evaluating different ontology alignment systems, especially those using machine learning techniques. We sincerely invite all kinds of OM systems to participate in our Bio-ML track. 
>
> In comparison with the 2022 version, we will make the following changes in 2023: 
>
> We will set up a new evaluation dataset to test large language models (LLMs) for equivalence matching. The idea is to rank a set of candidate target concepts for a given source concept, by exploring LLMs like Flan-T5 and the GPT-series and prompts. 
>
> We will cancel the validation set for simplicity. It will be merged with the testing set in the unsupervised setting, and merged with the training set in the semi-supervised setting.
>
> For subsumption matching, we will cancel the unsupervised setting, but keep the semi-supervised setting, for reducing the workload. 
>
> We will use modularization techniques to improve the to-be-matched ontologies that are extracted from the original large-scale ontologies according to the given ground truth mappings. This will keep more of the structure of the original ontology. 
>
> As in the last year, we encourage system submission via SEALS or HOBBIT, but can also allow direct result (mappings) submission, which will be marked in our results and reports. 
>
>
> Here are the temporary milestones: 
27th July: dataset release
31st August: system submission 
30th September: result release
14th October: report release
>
> Organizers: Yuan He, Jiaoyan Chen, Hang Dong, Ernesto Jiménez-Ruiz and Ian Horrocks

@cmungall can we assign this to anyone?

**Comment by cmungall** [comment]
Moving discussion to slack
